precompile/contracts/nativeminter/contract.go

```diff
@@ -117,7 +118,8 @@ func mintNativeCoin(accessibleState contract.AccessibleState, caller common.Addr
 		stateDB.CreateAccount(to)
 	}
 
-	stateDB.AddBalance(to, amount)
+	amountU256, _ := uint256.FromBig(amount) // XXX: should we check overflow?
```

Comment:
^

Reply:
This seems it would only check that if the `amount` exceeds uint256, but it does not actually check if the balance in `to` overflows after `amount`. so I don't think it's necesarry to check this.